testdata/project-v3-addon-and-grafana/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -8,6 +8,23 @@ metadata:
 spec:
   template:
     spec:
+      affinity:
```

Comment:
Same patch questions as above.

Reply:
Yes, in this case, we need to add it here.
This is the code implementation that will result to have the config done for the auth proxy
Note that this manifest is what does the patch to add the container to the Deployment where the Operator manager and the auth proxy are defined.

Reply:
Gotcha! Thanks for the explanation!